VelocidiSDK/VelocidiSDK/Match/VSDKMatchRequest.h

```diff
@@ -4,6 +4,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+  Class that wraps a NSURLMutableRequest with the necessary addons to make a match request to Velocidi's infrastructure
```

Comment:
Seems more accurate to me. What do you think?

(This suggestion also applies to other places)

```suggestion
  Class that wraps a NSURLMutableRequest with the necessary addons to make a match request to Velocidi's system
```